### PR TITLE
Adding some UI tweaks

### PR DESCRIPTION
**Pull request by kkatzen**
Consolidating some style variables. Creating placeholders for where final content can go later.

### CODE REVIEW

File: app/src/pages/LandingPage.tsx [hunk 1]
```diff
@@ -1,41 +1,92 @@
...
+              outcomes for vulnerable populations and to provide technical
+              assistance to health organizations, including state and local
+              jurisdictions’ COVID-19 responses.
+            </p>
+          </Grid>
+          <Grid item xs={6} className={styles.LandingSquare}>
+            <img
+              height="300px"
+              alt="placeholder"
+              src="https://upload.wikimedia.org/wikipedia/commons/5/56/David_Satcher_official_photo_portrait.jpg"
```
Comment (aaronsn) [inline app/src/pages/LandingPage.tsx:31]: I know this is just a placeholder until we figure things out, but if we're including pictures of people or other non-self-explanatory things, maybe we should have a small caption below the image?

Feel free to ignore this, we can always revisit this later

File: app/src/pages/LandingPage.tsx [hunk 2]
```diff
@@ -1,41 +1,92 @@
...
-      on five to ten objectives/thresholds determined by a national panel of
-      experts under our COVID-19 Health Equity Commission (CHEC). The project
-      creates a special track in HELEN focused on coordinating COVID-19
-      community engagement, education, training, information exchange and
-      dissemination, policy analysis, and advocacy.
+      <Grid container justify="space-around" className={styles.Grid}>
+        <Grid container justify="space-around">
+          <Grid item xs={6} className={styles.BlueLandingSquare}>
+            <Typography variant="h6" className={styles.HomeLogo}>
+              OUR INITIATIVE.
```
Comment (aaronsn) [inline app/src/pages/LandingPage.tsx:13]: Nit: I'd drop the periods on the end of these headers. I think it looks weird to have titles with periods

**Review by aaronsn — APPROVED** [review]
Nice, this looks great!

File: app/src/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import styles from "./Footer.module.scss";
+
+function Footer() {
+  return (
+    <div className={styles.Footer}>
+      <Grid container justify="space-around" className={styles.FooterGrid}>
+        <Grid item xs={5} className={styles.FooterColumn}>
```
[the remark this answers is not in the document]
Comment (kkatzen) [reply app/src/Footer.tsx:9]: Let's revisit when we get finalized mocks for this page!

File: app/src/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import styles from "./Footer.module.scss";
+
+function Footer() {
```
[the remark this answers is not in the document]
Comment (kkatzen) [reply app/src/Footer.tsx:5]: Good call! Done

**Pull request merged**